initializers/ip_addresses.yml

```diff
@@ -0,0 +1,13 @@
+#- address: 10.1.1.1/24
+#  device: server01
+#  interface: Loopback0
+#  status: Active
+#  vrf: vrf1
+#- address: 10.1.1.2/24
+#  device: server02
+#  interface: Vlan5
```

Comment:
This interface doesn't exist. I believe it should be the following:

```suggestion
#  interface: to-server01
```

Reply:
Fixed.